seed/factories/user.factory.ts

```diff
@@ -18,6 +18,7 @@ export const userFactory = generateFactory<User>(() => {
     password: generateRandomPassword(),
     phone: faker.string.numeric({ length: 10, allowLeadingZeros: false }),
     zip: faker.location.zipCode(),
+    roles: [faker.helpers.enumValue(UserRole)],
```

Comment:
Is there a way to limit this to roles that are _not_ `UserRole.admin`?

I don't think it's a good idea to be seeding instances of the app w/ potentially more than 1 known, consistent admin user..